app/views/applications/update_whitelisted_ip.scala.html

```diff
@@ -0,0 +1,65 @@
+@*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import include._
+@import model._
+@import utils.LoggedInUser
+@import model.Forms.WhitelistedIpForm
+
+@(application: ApplicationResponse, form: Form[Set[String]])(implicit request: Request[_], loggedInUser: LoggedInUser, messages: Messages, appConfig: config.AppConfig)
+
+@main(title = s"${appConfig.title} - Update whitelisted IPs") {
+
+<div class="ip-whitelist">
+    <nav class="breadcrumb-nav">
+        <ul>
+            <li class="breadcrumb-nav__item"><a href="@{routes.ApplicationController.applicationsPage(None)}">Applications</a></li>
+            <li class="breadcrumb-nav__item breadcrumb-nav__item--trail" aria-hidden="true"></li>
+            <li class="breadcrumb-nav__item"><a href="@{routes.ApplicationController.applicationPage(application.id.toString)}">@application.name</a></li>
+            <li class="breadcrumb-nav__item breadcrumb-nav__item--trail" aria-hidden="true"></li>
+            <li class="breadcrumb-nav__item">Update whitelisted IPs</li>
+        </ul>
+    </nav>
+
+    <h1 class="heading-xlarge">Update whitelisted IPs</h1>
+
+    @helper.form(action = controllers.routes.ApplicationController.updateWhitelistedIpAction(application.id.toString)) {
+        <p>Each IP address must be between the ranges: 0.0.0.0/24 - 255.255.255.255/32</p>
```

Comment:
That's a good point but I'm not sure it would make it easier to understand... That kind of gives you an indication of what range you are expected to use for each number.